sourcegraph/sourcegraph
# [Notebooks]: Remove notepad UI from notebooks feature

Closes https://github.com/sourcegraph/sourcegraph/issues/57996

## Test plan
- Check that notebooks still works as expected
- Check that search flow has no regressions




## Preview 🤩
[Preview Link](https://docs.sourcegraph.com/@vk/remove-notepad)
